### DISCUSSION TITLE
Add theme color meta tag (Feature request).

### DISCUSSION THREAD

**Discussion by george-bobby** [thedaviddias/Front-End-Checklist#466] [Ideas]
**Copy of #465 (Sorry I didn't saw the discussions tab)**
Not an issue, a feature request.
Please create a checkbox for adding theme color. Like,
```<meta name="theme-color" content="#fafafa">```

MDN Reference - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

**Comment by george-bobby** [comment]
Now on PR #467

**Reply by josephmoleka** [reply]
Hello brother
